apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx

```diff
@@ -7,9 +7,12 @@ import {
 import {
   desiredPositions,
   desiredPositionsIdToLabelMap,
+  locationStates,
+  locationStatesOptions,
   topSkills,
   topSkillsIdToLabelMap,
 } from '@talent-connect/talent-pool/config'
+import { useEffect } from 'react'
```

Comment:
Not needed? Protip: use the "organize imports" VsCode command

Reply:
We should definitely have an eslint rule for this.

Reply:
One of out several automations we should have in place. Can you look if there is a rule covering this?